components/aws/sagemaker/tests/integration_tests/resources/config/create-workteam/config.yaml

```diff
@@ -0,0 +1,10 @@
+PipelineDefinition: resources/definition/workteam_pipeline.py
+TestName: create-workteam
+Timeout: 3600
+Arguments:
+  region: ((REGION))
+  team_name: 'test-workteam'
```

Comment:
dont need quotes for values

Reply:
Yea, I realized this later based on your comment in the other PR. Will fix.

Reply:
Done to compare the minio read arn with the one created in sagemaker.